helpers/base.sh

```diff
@@ -314,45 +314,103 @@ cleanup() {
     set +x
 }
 
-# Creates an image which has two partitions. The first one is the boot
-# partition and the second one is the partition intended for the root
-# filesystem.
+# Creates an image with the specified number of partitions.
+# TODO: describe labels.
 # Globals:
 #     IMAGE
 # Arguments:
-#     Root filesystem size in bytes
+#     Alignment size in megabytes
+#     Partition1 size in megabytes
```

Comment:
It supposed to be called fstype:size
Let's put a note about fs type and default value for fs type